build/server/Dockerfile

```diff
@@ -5,7 +5,9 @@ WORKDIR /tmp/server
 
 COPY ./server /tmp/server
 
-RUN apt update -y
+RUN apt update -y && apt install -y python3-pip
+RUN python3 -m pip install rpi.gpio
```

Comment:
this is actually necessary since there is no `apt` install candidate for `python3-rpi.gpio`. Installing via pip works, as long as the image is run on a raspberry pi obviously